src/bin/pgcopydb/follow.c

```diff
@@ -1055,7 +1051,7 @@ follow_wait_subprocesses(StreamSpecs *specs)
 					}
 				}
 
-				log_level(logLevel,
+				log_level(exitedSuccessfully ? LOG_NOTICE : LOG_ERROR,
```

Comment:
That does NOT look like an improvement. When reading the code we need to focus first on the switch statement and which cases are being handled, only to then compute another condition for choosing the LOG level, where the flow of decision making was more obvious before.

I like the quote that sayth: “Programs must be written for people to read, and only incidentally for machines to execute.”  from Harold Abelson, Structure and Interpretation of Computer Programs.

Reply:
this is a fix for the `shadowed variable` warning which means there is another variable/function with the same name in the outer scope.